src/components/Column.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import * as Data from '../types';
+import CardCollapsed from './cards/collapsed/CardCollapsed';
+import Editable from './Editable';
+import * as DataService from '../helpers/data.service';
+import NewCardForm from './cards/collapsed/NewCardForm';
+
+export type ColumnProps = Data.Column;
+
+const CardHeader = styled.h2`
+  border-bottom: 1px solid grey;
+  padding-bottom: 1em;
+`;
+
+const ColumnBody = styled.div`
+  width: 20em;
+  background-color: #ececec;
+  border-radius: 1em;
+  border: 1px solid grey;
+`;
+
+const CardsContainer = styled.div`
+  background-color: #ECECEC;
+  padding: 0.3em 0.5em 0.3em;
+  border-radius: 1em;
+`;
+
+export default function Column(props: ColumnProps) {
+  const { id, title, cards } = props;
+
+  const changeTitle = useCallback((newTitle) => {
```

Comment:
> Naming your handler functions follows a similar naming convention to the one for event handler attributes. Simply remember that there is a flow being followed here when it comes to the cause and effect. Whereas an event attribute adheres to using the on prefix, we will adjust our handler naming convention so that it proceeds the event by adopting the handle prefix. Thus, the naming convention becomes handleSubjectEvent where Subject is the thing the handler is focused on and Event is the event taking place.

Reply:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
Обработчики: handleSubjectEvent
Обработчики props: onEvent
соответственно название обработчика: handleChangeColumnTitle